model_compression_toolkit/core/pytorch/utils.py

```diff
@@ -17,6 +17,8 @@
 from typing import Union
 from model_compression_toolkit.core.pytorch.constants import CUDA, CPU
 
+_device = CUDA if torch.cuda.is_available() else CPU
```

Comment:
What is the incentive for defining this global variable?
In my opinion, it is better to avoid such variables, and there is no need to enable device initialization from an outer scope.

Reply:
Agree, it's not pretty but it was just a quick way to enable multi gpu support across Mct.

Reply:
@haihabi 
Please take a look

Reply:
Hi @mikeseven,

Can you please modify this change to not use "global" in order to update the _device variable?
Besides that, I agree that this modification is useful.